core-ui/src/resources/Deployments/DeploymentCreate.js

```diff
@@ -69,6 +83,20 @@ export function DeploymentCreate({
       initialResource={initialDeployment}
       handleNameChange={handleNameChange}
     >
+      {isIstioFeatureOn ? (
+        <ResourceForm.FormField
+          simple
```

Comment:
is it supposed to be simple-only then?
i believe this flag should be removed completely